<launch>
  <node pkg = "iqr_ks106_uart" type = "dauxi_ks106_node" name = "dauxi_ks106" output = "screen">
    <param name = "port" type = "string" value = "/dev/dauxi_ks106"/>
    <!--<param name = "detect_model" type = "int" value = "2"/-->
    <param name = "baudrate" type = "int" value = "9600"/>
    <param name = "frequency" type = "int" value = "10"/>
    <param name = "topic" type = "string" value = "us"/>
    <param name = "us1_frame_id" type = "string" value = "ks106_us1"/>
    <param name = "us2_frame_id" type = "string" value = "ks106_us2"/>
    <param name = "us3_frame_id" type = "string" value = "ks106_us3"/>
    <param name = "us4_frame_id" type = "string" value = "ks106_us4"/>
  </node>
</launch>